libbeat/autodiscover/providers/kubernetes/service_test.go

```diff
@@ -279,7 +279,6 @@ func TestEmitEvent_Service(t *testing.T) {
 				"host":     "192.168.0.1",
 				"id":       uid,
 				"provider": UUID,
-				"port":     8080,
```

Comment:
Why is the port removed from service events?

Reply:
the code in the kubernetes.go which combines all events into a single event explicitly removes the port before it sends its to autodiscover.

Reply:
Yes, but is this intended? Wouldn't we want to send the service port in the event for its use in Heartbeat for example?

Reply:
Ok, @ChrsMark helped me to understand this. The port will be in the `configs` when needed, having it also here would be redundant.